packages/url_launcher/ios/Classes/UrlLauncherPlugin.m

```diff
@@ -81,7 +81,12 @@ - (void)handleMethodCall:(FlutterMethodCall *)call result:(FlutterResult)result
     if (useSafariVC.boolValue) {
       [self launchURLInVC:url result:result];
     } else {
-      [self launchURL:url result:result];
+        NSDictionary *options = nil;
+        if (@available(iOS 10.0, *)) {
```

Comment:
nit: I'd keep `handleMethodCall` focused on routing the calls and move this logic to launchUrl